projects/picker/src/lib/date-time/adapter/native-date-time-adapter.class.ts

```diff
@@ -26,6 +26,9 @@ const ISO_8601_REGEX = /^\d{4}-\d{2}-\d{2}(?:T\d{2}:\d{2}:\d{2}(?:\.\d+)?(?:Z|(?
 
 @Injectable()
 export class NativeDateTimeAdapter extends DateTimeAdapter<Date> {
+ 	protected firstMonthOfTheYear: number = 0;
```

Comment:
@Venators  for all the time adapters, you've incorrectly extended the base class, causing a build error for production. 

`DateTimeAdapter` has `firstMonthOfTheYear` and `firstDayOfTheWeek` public, but the derived adapters are changing the access modifiers to something more strict (protected).

Reply:
Hi @danielmoncada - my apologies for that mix-up.
The code should be corrected now with the matching & appropriate access modifiers.